pkgs/tools/misc/tmux-sessionizer/default.nix

```diff
@@ -19,16 +20,19 @@ in rustPlatform.buildRustPackage {
   src = fetchFromGitHub {
     owner = "jrmoulton";
     repo = name;
-    rev = "v${version}";
-    hash = "sha256-8RQ67v2Cw681zikxYnq0Pb2ybh26w8mUbHKAC4TjYWA=";
+    rev = "79ab43a4087aa7e4e865cab6a181dfd24c6e7a90";
+    hash = "sha256-gzbCeNZML2ygIy/H3uT9apahqI+4hmrTwgXvcZq4Xog=";
   };
 
-  cargoHash = "sha256-ZOWoUBna8U0A/sYwXMf4Z7Vi+KqM7VinWhmtO8Q0HtU=";
+  cargoHash = "sha256-7pjJG55O2MSCFBOlJk3tKk3l+RC08L9kv8GFEwsyNzU=";
 
   passthru.tests.version = testers.testVersion {
```

Comment:
We will probably need to disable the version test for now.

Reply:
I suppose we could set the expected version manually?

```nix
passthru.tests.version = testers.testVersion {
  package = tmux-sessionizer;
  version = "0.4.0";
};
```